src/commctl/cli.py

```diff
@@ -633,6 +636,9 @@ def add_cluster_commands(argument_parser):
     # Sub-command: cluster create
     verb_parser = subject_subparser.add_parser('create')
     verb_parser.required = True
+    verb_parser.add_argument(
+        'type', help='Type of the cluster',
+        choices=('kubernetes', 'host_only'))
```

Comment:
I'll make it ``--type`` right now but I was planning on [requiring a type upon first release](http://commissaire.readthedocs.io/en/latest/endpoints.html#put). Does that sound OK?